README.md

```diff
@@ -42,6 +42,15 @@ The planning framework (Tesseract) was designed to be light weight, limiting the
 * [Doxygen](https://ros-industrial-consortium.github.io/tesseract/)
 * [Benchmark](https://ros-industrial-consortium.github.io/tesseract/dev/bench)
 
+## Evolution
+
+![](gh_pages/_static/tesseract_evolution.gif)
```

Comment:
While I think this is neat to see, I'm not sure we want to add the gif itself to the git history. Its an 87 MB binary file that will probably change every few months. Can we use the github LFS system for this or do we not have that on the this organization?

Reply:
Looks good.